lib/resources.rb

```diff
@@ -64,6 +65,13 @@ def link_header(rels)
         lines
       end
 
+      def headers_path(bulk, r_4)
+        path = r_4 ? '../../../' : '../../'
+        path = bulk ? '../../bulk-data' : path
```

Comment:
do we need to have the bulk-data hard coded can't we use the relative path?

Reply:
rubocop throws the following warning since it is a numbered variable, hence why I added the underscore:
`Naming/VariableNumber: Use snake_case for variable numbers.`